tests/ops/api/v1/endpoints/test_user_endpoints.py

```diff
@@ -218,35 +218,41 @@ def test_create_user_wrong_scope(self, url, api_client, generate_auth_header, db
         response = api_client.delete(url, headers=auth_header)
         assert HTTP_403_FORBIDDEN == response.status_code
 
-    def test_delete_user_not_admin_root_or_self(
-        self, url, api_client, db, generate_auth_header, user
-    ):
-        auth_header = generate_auth_header([USER_DELETE])
-        response = api_client.delete(url, headers=auth_header)
-        assert HTTP_403_FORBIDDEN == response.status_code
-
     def test_delete_nonexistent_user(self, api_client, db, generate_auth_header, user):
         auth_header = generate_auth_header([USER_DELETE])
         url = f"{V1_URL_PREFIX}{USERS}/nonexistent_user"
         response = api_client.delete(url, headers=auth_header)
         assert HTTP_404_NOT_FOUND == response.status_code
 
-    def test_delete_self(self, api_client, db, generate_auth_header):
+    def test_delete_user(self, api_client, db, generate_auth_header):
```

Comment:
We still want a test for the case where a user deletes themselves. Consider moving these changes into a new test entirely, that tests explicitly what we want to test here (that a user can be deleted by another with the correct scope).

Reply:
As it stands, the user needs `user:delete` in order to delete themselves. This makes sense to me.